### ISSUE TITLE
Spotify often starts muted

### ISSUE THREAD

**Issue by Vadammt** [Xeroday/Spotify-Ad-Blocker#174]
### Problem
Basically everything works fine. But from time to time Spotify starts up muted and I manually have to unmute it in the Windows Volume Mixer.

### Cause
I couldn't figure out how to reproduce for sure! This error might be caused by closing Spotify / EZBlocker while it was playing ads.

### Solution 
**Just a wild guess** Run "is playing ads"-check after startup of Spotify and if applicable mute / unmute it.

---

Spotify v1.0.89.313.g34a58dea
EZBlocker 1.7.2.5

**Comment by thatguy344** [comment]
Wow, this is some annoying bug. I thought those devs had outsmarted Ezblocker yet again or something lol 

Also noticed that with the new client version it sometimes randomly closes itself as soon as Ezblocker becomes active for some reason. Also sad to see that most of the new banner ads baked into the client (such as the one below the player or the one that takes up the whole screen when you un-minimize the client again, not sure if that's a localized thing though) cannot be blocked by default. If you leave the host file blocks in what happens is that most of the basic functionality such as artist radio aggregation and so on stops working aswell. Oh yeah and whereas before ezblocker was able to skip ads entirely they've now managed to brute-force them in so that they can only be muted and have to be sat out... I'm almost kinda surprised it took them this long

**Comment by WorkingRobot** [comment]
You can open the volume mixer and unmute spotify.

**Comment by Vadammt** [comment]
Yes, but this is rather a workaround than a solution.

**Comment by x0uter** [comment]
I have the same problem. 
Auto Unmute Spotify volume was a good feature.

**Comment by CzBuCHi** [comment]
Just stumbled uppon this when creating new issue....

What about adding (un)mute spotify button / tray icon context menu / keyboard shortcut? - it will not fix real issue, but simplify current workaround ... (i may create PR)
